docs/cudf/source/user_guide/guide-to-udfs.ipynb

```diff
@@ -2,15 +2,16 @@
  "cells": [
```

Comment:
<p>This looks to be an issue with the way ReviewNB diffs things. There's no <code>====================================</code> in the source. Ditto in many places below.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cudf/pull/10691/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>